blog post about translating seagl to virtual (#344)
SeaGL/seagl.github.io: _posts/2022-10-07-Building-a-virtual-SeaGL.md

---
layout: post
title: 'Building a virtual SeaGL'
status: publish
type: post
published: true
categories: news
tags: '2022'
---

> As SeaGL turns ten, relying on the virtual format for a third year, it's time to talk foundations. How does a Free Software conference approach the transition from in-person to Internet?

Author: I rather like this play on words :P

First, we considered what we like about _our_ conferences:

## They're accessible

> We are trying to build an open and welcoming community, not an exclusive club. To accomplish this, there is no fee to attend and we minimize the number of barriers to entry. Additionally, we have been hosted in a central location that is near transit lines, eateries, and lodging. In recent years, we've also been able to provide childcare for those too young to attend.

Reviewer: ```suggestion
We are trying to build an open and welcoming community, not an exclusive club. To accomplish this, there is no fee to attend and we minimize the number of barriers to entry. Additionally, we have been hosted in a central location that is near transit lines, eateries, and lodging. In recent years, we've also been able to provide childcare for those who can't attend with their children.
```

The old wording here makes it sound like childcare is targeted at children when really it's a service for the parents

## They respect privacy

> Avoiding the collection of personal data can be tricky, our primary solution has been to not require registration. While this may cause us headaches when ordering coffee or pizza, it has been one of SeaGL's core values from the beginning. Relatedly, our mailing lists are opt-in and we don't share them with sponsors.

Reviewer: ```suggestion
Avoiding the collection of personal data can be tricky, and our primary solution has been to not require registration. While this may cause us headaches when ordering coffee or pizza, it has been one of SeaGL's core values from the beginning. Our mailing lists are also opt-in and we don't share their membership lists with sponsors.
```

## They're social

> People attend conferences for much more than scheduled programming. A term that encompasses this well is "hallway track". We've always built in dedicated social time, whether through extended breaks between talks or socializing opportunities such as TeaGL.

Reviewer: ```suggestion
People attend conferences for much more than scheduled programming - a concept commonly referred to as the "hallway track". We've always built in dedicated social time, whether through extended breaks between talks or socializing opportunities such as TeaGL.
```

## They're civil

> Our conference attracts people from many different backgrounds, holding many contrasting views. However, despite having developed a robust Code of Conduct, violations have been rare and well-managed. Our attendees tend to bring an open ethos, leading to understanding and respect.

Reviewer: Not quite sure how to address this but I don't like "despite". It's probably _because_ of our robust CoC that we don't have problems. It's also awkward grammatically IMO to say "despite having a robust CoC, [CoC] violations have been rare..."

---

> Now, how do we translate these values to a virtual setting? (aside from the obvious: avoiding non-free software)

Reviewer: ```suggestion
Now, how do we translate these values to a virtual setting? (Aside from the obvious: avoiding non-free software!)
```

## Access

Going remote has at least one huge advantage over the physical space; people can participate from all over the planet! Whether attending, presenting, sponsoring, or volunteering, we have spread our wings and welcomed folks from at least four continents.

> Aside from location, we have also been able to provide resources for disabled participants. For instance, talks now have closed captioning and are recorded for later review. Additionally, people can now attend without the strains of transportation.

Reviewer: Nit: we've recorded talks since before the pandemic

Reviewer: Do we want to mention transportation again here? The previous line kinda covers it already

Author: Made some changes to address these, but happy for further suggestions.

And of course, our enthusiastic team of volunteers has been working hard to keep the event cost-free to attend.

## Privacy

While metadata is unavoidable with networked devices, we have continued to ask ourselves tough questions. How little tracking can we get away with? Is a virtual conference possible without user accounts? Must privacy be compromised for security?

> Perfect answers may not exist for each of these, but we feel like we've come up with some darned good ones that are continually refined. The last one is especially tricky given that we are exposed to more potential abuse online.

Reviewer: ```suggestion
Perfect answers may not exist for each of these, but we feel like we've come up with some darned good ones that we continue to refine even further. The last one is especially tricky given that we are exposed to more potential abuse online.
```

Reviewer: (Avoiding passive voice here)

> Perfect answers may not exist for each of these, but we feel like we've come up with some darned good ones that are continually refined. The last one is especially tricky given that we are exposed to more potential abuse online.

Reviewer: ```suggestion
Perfect answers may not exist for each of these, but we feel like we've come up with some darned good ones that are continually refined. The last question about privacy vs. security is especially tricky given that we are exposed to more potential abuse online.
```

## Socially

> Interacting through text and video is just not the same as in-person. Text is clumsy and can easily lead to misunderstandings. Video calls may introduce latency and exclude the space that each person is in. Being behind a screen it is all too easy to tune out or multi-task.

Reviewer: > and exclude the space that each person is in.

Not sure what this means

Author: this is reference to the "tunnel vision" effect of only seeing the head and shoulders of a person. You don't know if there's something else trying to get their attention in the background for instance. Unsure if it describes it well enough but happy to make modifications.

> Interacting through text and video is just not the same as in-person. Text is clumsy and can easily lead to misunderstandings. Video calls may introduce latency and exclude the space that each person is in. Being behind a screen it is all too easy to tune out or multi-task.

Reviewer: ```suggestion
Interacting through text and video is just not the same as in-person. Text is clumsy and can easily lead to misunderstandings. Video calls may introduce latency and exclude the space that each person is in. Being behind a screen makes it all too easy to tune out or multi-task.
```

What's more, it's 2022 and we're all tired of virtual everything...

> That said, these are all just challenges and considerations that we've taken into account. Again, our solutions may not be perfect, but we've strived to prioritize the social conference experience.

Reviewer: This sentence doesn't really seem to be connected to the previous ones. Maybe:

```suggestion
But these are all challenges we believe we can meaningfully improve. Again, our solutions may not be perfect, but we've strived to prioritize the social conference experience.
```

## Civility

> For some reason, people tend towards being more mean over the Internet. Misunderstandings may escalate and, for a technical audience, abuse tools become a real concern. Traditional moderation methods are still relevant, but the speed and scale of a response may be much more challenging.

Reviewer: ```suggestion
For some reason, people tend towards being more mean over the Internet. Misunderstandings may escalate and, especially for a technical audience, abuse tools become a real concern. Traditional moderation methods are still relevant, but the speed and scale of a response may be much more challenging.
```

> For some reason, people tend towards being more mean over the Internet. Misunderstandings may escalate and, for a technical audience, abuse tools become a real concern. Traditional moderation methods are still relevant, but the speed and scale of a response may be much more challenging.

Reviewer: ```suggestion
For some reason, people tend towards being more mean over the Internet. Misunderstandings may escalate and, for a technical audience, abuse tools become a real concern. Traditional moderation methods are still relevant but the speed and scale of a response may be much more challenging.
```

Too many commas in this paragraph IMO - makes me feel like I'm stumbling through the sentences cause of all the pauses

## Community

In-person conferences are very time-bound. Participants may look forward to the next one, but rarely do they stay connected with each other in-between. The virtual environment provides new opportunities to bring people into a year-round space, holding both the conference and free software movement together.

---

> Now... onto the tech!

Reviewer: This transition is jarring. Maybe:

```suggestion
So... what's the tech stack that powers all these values?
```

Or something like that. Something to connect it back to the previous section

> For our first virtual conference, we mashed together IRC, Kiwi, BigBlueButton, and Restreamer. It was a hit and the next year we expanded from IRC by bridging into Matrix. Our excitement has only grown about the Matrix project and its goal of creating an open network for secure, decentralized communication. Thus, this year we've decided to double-down and run the event primarily on Matrix.

Reviewer: Maybe some links for those who don't know what these things are? IRC seems OK to assume but the rest seem a little too specific to leave to chance

With November rapidly approaching, we're assembling the pieces yet again! During this process we are trying to responsibly balance the competing goals and questions outlined above. Some of these are fundamentally at odds, some we might have a cleaver solution for, and some are stuck competing for volunteer bandwidth...

> Does this vision excite you? We welcome additional wings and beaks! Please contact participate@seagl.org to volunteer today!

Reviewer: ```suggestion
Does this vision excite you? Want to hear more details about how we're solving these problems? We welcome additional wings and beaks! Please contact participate@seagl.org to volunteer today!
```

I added this because I feel like the blog post is a little sparse on actual details about how we're "building a virtual SeaGL". This sentence kills two birds with one stone: "if you want to get the full story, you'll have to show up to our volunteer spaces..."
